nestkernel/conn_builder.cpp

```diff
@@ -337,6 +337,61 @@ nest::ConnBuilder::single_connect_( index sgid,
   }
 }
 
+inline void
+nest::ConnBuilder::skip_conn_parameter_( thread target_thread, librandom::RngPtr& rng )
+{
+  if ( param_dicts_.empty() ) // indicates we have no synapse params
+  {
+    if ( default_weight_ )
+      delay_->value_double( target_thread, rng );
+    else if ( ( !default_weight_and_delay_ ) && ( !default_weight_ ) )
```

Comment:
I spent several minutes wrapping my poor brain around the logic of the Boolean and the variable names (yes, I invented them ...). Can we find a better set of names?